tracer/src/Datadog.Trace/Logging/TracerFlare/TracerFlareManager.cs

```diff
@@ -34,6 +33,8 @@ internal class TracerFlareManager : ITracerFlareManager
     private readonly IRcmSubscriptionManager _subscriptionManager;
     private readonly ITelemetryController _telemetryController;
     private readonly TracerFlareApi _flareApi;
+
+    private string? _debugEnabledConfigPath;
```

Comment:
Do we need to keep track of potentially more than one of these? 🤔 Would seem a bit odd, but given we're not 100% on the expected behaviour 🤷‍♂️

Reply:
I thought about it and eventually decided that "meh"